apps/commudle-admin/src/app/feature-modules/community-control-panel/components/community-members/community-members.component.ts

```diff
@@ -7,14 +7,16 @@ import { EUserRoles } from 'apps/shared-models/enums/user_roles.enum';
 import { IUser } from 'apps/shared-models/user.model';
 import { IUserRolesUser } from 'apps/shared-models/user_roles_user.model';
 import { debounceTime, filter, map, switchMap } from 'rxjs/operators';
+import { ICommunity } from 'apps/shared-models/community.model';
+import { Subject, takeUntil, Subscription } from 'rxjs';
+import { SeoService } from 'apps/shared-services/seo.service';
```

Comment:
let's use @commmudle/shared-models for ICommunity and SeoService from @commudle/shared-service